### ISSUE TITLE
18.1.3 snap blank white 

### ISSUE THREAD

**Issue by wiryonolau** [jgraph/drawio-desktop#1029]
### Preflight Checklist
* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

You must agree to search and the code of conduct. You must fill in this entire template. If you delete part/all or miss parts out your issue will be closed.

**Describe the bug**
Start drawio, then only blank white window appear

**To Reproduce**
Run draw.io as usual

**Expected behavior**
Start normally 

**Screenshots**
![Screenshot_2022-06-03_12-23-00](https://user-images.githubusercontent.com/9910819/171791591-a234c61d-dab8-47ea-bcd6-f46897a5e6bc.png)

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version 18.1.3

**Desktop (please complete the following information):**
 - OS: Test on Ubuntu 20.04, Debian 11 and run using snap
 
**Additional context**
Start from console got below error
```bash
[6706:0603/122009.215794:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.
WARNING: Kernel has no file descriptor comparison support: Operation not permitted
12:20:09.369 › Checking for update
[6732:0603/122009.442562:ERROR:platform_shared_memory_region_posix.cc(250)] Creating shared memory in /dev/shm/.org.chromium.Chromium.PNlH9g failed: Operation not permitted (1)
[6732:0603/122009.442748:ERROR:platform_shared_memory_region_posix.cc(253)] Unable to access(W_OK|X_OK) /dev/shm: Operation not permitted (1)
[6732:0603/122009.442822:FATAL:platform_shared_memory_region_posix.cc(255)] This is frequently caused by incorrect permissions on /dev/shm.  Try 'sudo chmod 1777 /dev/shm' to fix.
12:20:11.635 › Update for version 18.1.3 is not available (latest version: 18.1.3, downgrade is disallowed).
```

**Comment by lebigsquare** [comment]
Same as above : issue has been running as described for the past 10-15 days.

**Comment by davidjgraph** [comment]
Duplicate of #1018 . We're likely going to stop the snap build and suggest to use .deb instead.

**Comment by cbartz** [comment]
For me this is fixed with 19.0.0 and I am still using snap.
